docs/guide/authjs/quick-start.md

```diff
@@ -0,0 +1,154 @@
+# AuthJS Quickstart
+
+This guide is for setting up `@sidebase/nuxt-auth` with the AuthJS Provider, which is best suited for plug-and-play OAuth for established oauth-providers or magic-url based sign-ins.
+
+## Installation
+
+If you want to use the AuthJS provider, you have to install `next-auth`. With all package managers except npm you must manually install the peer dependency alongside nuxt-auth:
+
+::: code-group
+
+```bash [pnpm]
+pnpm i next-auth@4.21.1
+```
+
+```bash [yarn]
+yarn add next-auth@4.21.1
+```
+
+::: warning
+Due to a breaking change in NextAuth, nuxt-auth is only compoatible with NextAuth versions under v4.23.0. We recommend pinning the version to `4.21.1`. Read more [here](https://github.com/sidebase/nuxt-auth/issues/514).
+:::
+
+## Configuration
+
+After installing `@sidebase/nuxt-auth` and `next-auth`, you can now configure NuxtAuth to use the AuthJS provider. Inside your `nuxt.config.ts` add the following configuration:
+
+```ts
+export default defineNuxtConfig({
+    modules: ['@sidebase/nuxt-auth'],
+    auth: {
+        provider: {
+            type: 'authjs'
+        }
+    }
+})
+```
+
+You can also configure AuthJS specific options inside the `nuxt.config.ts`, in addition to the main [module configurations](/guide/application-side/configuration).
+
+### `trustHost`
+
+- **Type**: `boolean`
+- **Default**: `false`
+
+If set to `true`, `authjs` will use either the `x-forwarded-host` or `host` headers instead of `auth.baseURL`. Make sure that reading `x-forwarded-host` on your hosting platform can be trusted.
+
+:::warning
+**This is an advanced option.** Advanced options are passed the same way as basic options, but **may have complex implications** or side effects. You should try to avoid using advanced options unless you are very comfortable using them.
+:::
+
+### `defaultProvider`
+
+- **Type**: `SupportedProviders`
+- **Default**: `undefined`
+
+Select the default-provider to use when `signIn` is called. Setting this here will also effect the global middleware behavior: E.g., when you set it to `github` and the user is unauthorized, they will be directly forwarded to the Github OAuth page instead of seeing the app-login page.
```

Comment:
```suggestion
Select the default-provider to use when `signIn` is called. Setting this here will also affect the global middleware behavior. For instance, when you set it to `github` and the user is unauthorized, they will be directly forwarded to the Github OAuth page instead of seeing the app-login page.
```